candidate_base.go

```diff
@@ -206,57 +205,53 @@ func (c *candidateBase) start(a *Agent, conn net.PacketConn, initializedCh <-cha
 }
 
 func (c *candidateBase) recvLoop(initializedCh <-chan struct{}) {
-	defer func() {
-		close(c.closedCh)
-	}()
+	defer close(c.closedCh)
```

Comment:
:+1: love cleanup like this :)

Reply:
😁 I was really wondering how this happened to get into pion/ice originally and questioned my Go skills...